packages/core/src/shared/clients/docdbClient.ts

```diff
@@ -3,14 +3,33 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import * as vscode from 'vscode'
-import _ from 'lodash'
-import { AWSError, DocDB, DocDBElastic } from 'aws-sdk'
-import { getLogger } from '../logger'
+import globals from '../extensionGlobals'
 import { InterfaceNoSymbol } from '../utilities/tsUtils'
+import { DocDBClient, DocDBClientConfig, DBCluster, DescribeDBClustersCommand } from '@aws-sdk/client-docdb'
 
 export type DocumentDBClient = InterfaceNoSymbol<DefaultDocumentDBClient>
 
 export class DefaultDocumentDBClient {
     public constructor(public readonly regionCode: string) {}
+
+    public async getClient(): Promise<DocDBClient> {
+        const credentials = await globals.awsContext.getCredentials()
+        const config: DocDBClientConfig = {
+            credentials: credentials,
+            region: this.regionCode,
+        }
+        return new DocDBClient(config)
```

Comment:
Shall we do an error handling here, ensuring the client can be created and credentials and regions are populated?

Reply:
Error handling is inconsistent over this toolkit.
Options to consider
log error and re-throw same exception (LambdaClient is example) 
throw `ToolkitError` (ec2Client is example)
do not handle exceptions (ecsClient)

I think best option to log error and throw `ToolkitError`